### PR TITLE
Draft of Qrack blog post

### PR DESCRIPTION
**Pull request by WrathfulSpatula**
This is a draft of a new blog post about Qrack's recent publication in IEEE Quantum Week '23 proceedings. Please, any and all feedback is welcome!

### CODE REVIEW

File: src/content/blog/qrack_report.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
+---
+title: "Qrack and PyQrack publish their first formal report"
```
Comment (nathanshammah) [inline src/content/blog/qrack_report.md:2]: What about "Simulating 54 qubits with Qrack – on a single GPU"?

File: src/content/blog/qrack_report.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+
+At a high level, Qrack can make many of the same kinds of general performance claims as conventional tensor network approaches: for special cases of low-entanglement or "near-Clifford" simulation, it can often support hundreds or thousands of qubits in a single quantum circuit. Since Qrack robustly supports interoperability with major conventional tensor network software, it's also incredibly easy to combine techniques in Qrack with use of those more popular back ends. However, we think many users would be surprised at how much equivalent functionality, compared to tensor networks, is already covered by Qrack, whether through relatively "novel" simulation techniques.
```
[the remark this answers is not in the document]
Comment (WrathfulSpatula) [reply src/content/blog/qrack_report.md:30]: My hesitation, here, is that the paragraph is _not_ about approximation techniques, specifically. In "low-entanglement" and/or "near-Clifford" conditions, Qrack can often do hundreds or thousands of qubits completely in the ideal, without approximation.

[the remark this answers is not in the document]
Comment (nathanshammah) [reply src/content/blog/qrack_report.md:36]: @WrathfulSpatula I changed "approximate methods" with "simulation methods"

**Review by nathanshammah — APPROVED** [review]
@WrathfulSpatula looks great! I just included the code snippets and figure suggestions. Up to you to include them. It may be too much work, your call, in that case feel free to merge!

**Comment by nathanshammah** [comment]
Oh, I didn't see the figures, those are great!

**Pull request merged**